src/components/navBar/navBar.tsx

```diff
@@ -1,66 +1,101 @@
 import './navBar.css'
 
+import { useMediaQuery } from '@mui/material'
 import AppBar from '@mui/material/AppBar'
-import IconButton from '@mui/material/IconButton'
+import BottomNavigation from '@mui/material/BottomNavigation'
+import BottomNavigationAction from '@mui/material/BottomNavigationAction'
+import Box from '@mui/material/Box'
 import { useTheme } from '@mui/material/styles'
 import Toolbar from '@mui/material/Toolbar'
 import type { ReactNode } from 'react'
 import React from 'react'
 
-interface NavBarProps {
-  /** Desktop version: Logo is displayed in the top left corner. Tablet and mobile versions: position will be different if you add the drawer icons. */
-  logo: ReactNode
-  /** Icon for the button that opens/closes the left drawer. (Will only be shown in mobile and tablet versions.) */
-  leftDrawerIcon?: ReactNode
-  /** Icon for the button that opens/closes the right drawer. (Will only be shown in mobile and tablet versions.)*/
-  rightDrawerIcon?: ReactNode
-  /** Aria label for the left drawer button. */
-  leftDrawerAriaLabel?: string
-  /** Aria label for the right drawer button. */
-  rightDrawerAriaLabel?: string
-  /** Function for toggling the open/closed state of the left drawer. */
-  handleLeftDrawerToggle?: () => void
-  /** Function for toggling the open/closed state of the right drawer. */
-  handleRightDrawerToggle?: () => void
+export interface BottomNavBarItem {
+  label: string
+  onClick: () => void
+  icon: ReactNode
+}
+
+export interface TopNavBarItem {
+  node: ReactNode
+}
+
+export interface NavBarProps {
+  /**
+   * The logo to be displayed in the navigation bar.
+   */
+  logo?: ReactNode
+
+  /**
+   * An array of items to be displayed in the top navigation bar. We recommend at most 3 items.
+   */
+  topNavBarItems?: TopNavBarItem[]
+
+  /**
+   * An array of items to be displayed in the bottom navigation bar on mobile devices. We recommend at most 3 items.
+   */
+  bottomNavBarItems?: BottomNavBarItem[]
```

Comment:
```suggestion
export interface NavBarProps {
  /** The logo to be displayed in the navigation bar. */
  logo?: ReactNode
  /** An array of items to be displayed in the top navigation bar. We recommend at most 3 items. */
  topNavBarItems?: TopNavBarItem[]
  /** An array of items to be displayed in the bottom navigation bar on mobile devices. We recommend at most 3 items.  */
  bottomNavBarItems?: BottomNavBarItem[]
```

Reply:
I'm wondering if logo should be mandatory

Reply:
> I'm wondering if logo should be mandatory

Yeah I wasn't too sure about this.